### ISSUE TITLE
DOM driver doesn't receive events from inside the shadow dom

### ISSUE THREAD

**Issue by tusharmath** [cyclejs/cyclejs#412]
**Code to reproduce the issue:**

``` js

// DOM element needs to have a parent other wise snabbdom won't render
function createWrapperNode () {
  const wrapper = document.createElement('x-wrapper')
  const node = document.createElement('x-node')
  wrapper.appendChild(node)
  return {node, wrapper}
}

// Lifecycle method — http://www.html5rocks.com/en/tutorials/webcomponents/customelements/
function createdCallback () {
  const shadowRoot = this.createShadowRoot()
  const {parent, node} = createWrapperNode()
  this.__vNode = node
  shadowRoot.appendChild(parent)
  const drivers = {
    DOM: makeDOMDriver(this.__vNode)
  }
}
```

**Expected behavior:**

``` js
DOM.select('.some-selector').events('click') // should fire
```

**Actual behavior:**
No event is fired.

**Versions of packages used:**

``` json
    "@cycle/dom": "^12.2.2",
    "@cycle/rx-run": "^7.0.1",
    "@cycle/xstream-run": "^3.1.0",

```

Creating web components using cyclejs helps in having a consistent paradigm across the application. I have created a special driver to handle attribute changes but need help with the DOM driver.

**Comment by tusharmath** [comment]
@staltz what are your thoughts on this?

**Comment by tusharmath** [comment]
Sorry, this wasn't a real issue.

**Comment by tusharmath** [comment]
I was able to reproduce this here — https://esnextb.in/?gist=5e6c955cb6072800b3a73f7abfe56fce

**Comment by staltz** [comment]
Released in Cycle DOM v14.2 thanks to @ajmchambers.
